src/Controls/src/Core/Page/Page.cs

```diff
@@ -234,7 +310,18 @@ public Task<bool> DisplayAlert(string title, string message, string accept, stri
 			return args.Result.Task;
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Page.xml" path="//Member[@MemberName='DisplayPromptAsync'][2]/Docs/*" />
+		/// <summary>
+		/// Displays a prompt dialog to the application user with the intent to capture a single string value.
+		/// </summary>
+		/// <param name="title">The title of the prompt dialog.</param>
```

Comment:
Can any of these params be `null`? I'm curious because the previous `DisplayX` methods specified that a parameter can be null to _not_ display certain parts of the alert.

Reply:
Yeah those were all taken directly from the external XML docs. I'm not sure I like it yet. It's very clear, but how far do you want to take it? 😅

I've removed the "Must not be null" from the other APIs and only specified whenever something can be made null to actually hide buttons or options and applied the same to here.

Reply:
Hmmmm to be honest I'm not 100% sure how to approach this one. Tagging @davidbritch to save us.

Reply:
I think it's fine as is, bearing in mind resource constraints. It could be that null behaviour for each parameter is different on different platforms, which'll require a bit of work to provide that level of detail.